aspnetcore/test/http-files.md

```diff
@@ -130,6 +131,69 @@ Variables can be defined using values of other variables that were defined earli
 GET https://{{host}}/api/search/tool
 ```
 
+### Handling null values
+
+When working with variables in `.http` files, you might need to handle `null` values in requests or responses. Here's how to work with null values:
+
+#### Setting a variable to null
+
+To explicitly set a variable to `null` in your request body:
```

Comment:
This is fine and works.
To be more elaborate we can also mention that this works because `null` is a valid JSON value.
But I agree that this adds more verbosity, which isn't needed.